app/scripts/modules/core/src/search/widgets/Filters.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import autoBindMethods from 'class-autobind-decorator';
+
+import { Key } from '@spinnaker/core';
```

Comment:
Within `core`, imports should be relative or with `core` as the root, e.g. `core/widgets/keys`